BlueprintUI/Sources/Environment/Keys/CalendarKey.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+extension Environment {
+    private enum CalendarKey: EnvironmentKey {
+        static var defaultValue: Calendar {
+            Calendar.current
```

Comment:
There's an `autoupdatingCurrent` getter that updates if the calendar changes. Not sure we should use it, since you can end up using two different calendars if it changes and your UI doesn't update itself - but might be good to document that this defaults to just `current`

Reply:
Good catch! I actually checked on this, and chose it because SwiftUI's calendar property defaults to `current`, not `autoupdatingCurrent`. I'll add a comment about the default.